### ISSUE TITLE
Publisher stats seem to include editions not by that publisher

### ISSUE THREAD

**Issue by agmckee** [internetarchive/openlibrary#7731]
I visited the page of a publisher that's no longer active and noticed the stats stated edition range from 1835-2023. Curious what could have been published [after the firm was voluntarily dissolved last year](https://find-and-update.company-information.service.gov.uk/company/00386574/filing-history/MzM0OTUyMzAyOWFkaXF6a2N4/document?format=pdf&download=0), and with the intention of fixing a catalogue error if relevant, I found that it's actually showing a work that did have an edition published by Futura in the past but the 2023 edition is not by this publisher. Thus it seems there's a bit of a logical error in how the publication range is calculated. I suspect the 1835 is also erroneous, as Futura was founded in the 1940s but presumably published an edition of an older work.

### Evidence / Screenshot (if possible)

![open-library-publisher-futura-includes-edition-not-by-futura](https://user-images.githubusercontent.com/722096/227734798-f7b0687d-afa2-42ff-ac66-bef49c4a2e0b.png)

### Relevant url?
[https://openlibrary.org/publishers/Futura](https://openlibrary.org/publishers/Futura)

### Steps to Reproduce
1. Go to [https://openlibrary.org/publishers/Futura](https://openlibrary.org/publishers/Futura)
2. Observe publication range, in this instance "Published between 1835 & 2023"
3. Click on the latest year (2023) on the graph

 - Actual:
   - The page will load the _work_ https://openlibrary.org/works/OL893503W/The_Jesus_Incident
     - The edition shown (for me, at least) is the GOLLANCZ edition from 2000.
     - The edition published in 2023 is a different edition, not published by Futura or GOLLANCZ
* Expected:
  - There should not be anything for 2023, as this publisher cannot have published this year.
   - The page would only show published editions by this publisher, not editions by other publishers

### Details

- **Logged in (Y/N)?** Yes
- **Browser type/version?** Firefox, 102.9 ESR
- **Operating system?** Debian
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

I propose that the publisher page should show the editions that this specific publisher published, which would make sense in context. The data should be available to allow for this, from my understanding of how OL stores this information.

**Comment by mekarpeles** [comment]
Thank you for submitting! I wonder if the query for the graph needs updating and if this may be related to #7718 under the hood.

**Comment by agmckee** [comment]
I had a look into it and it seems it loads a list of works (it uses subject search). The result set is just a list of works that has ever had an edition by this publisher. Naturally this works fine for publisher pages with works with the same publisher for all editions.

I investigated the search methods parameters to see if there was a straightforward way to filter down the result set to editions with the same publisher (i.e. the publisher that is the subject of the page) but couldn't find a simple way. I noticed that author pages work very differently as do most other things.

Publisher page is currently implemented like a category page. Categories are a property of works while publisher is an edition level property. That seems to be the core of the issue.

**Comment by bicolino34** [comment]
Seems to be a duplicate of #158 
There are some issues about duplicate publishers and one of the proposal is to create an ID for publishers, which could help in resolving that issue. See #6570 #372 #200

**Comment by tfmorris** [comment]
Yes, a duplicate of #158. Now that editions are indexed, it should be possible to fix.

**Comment by agmckee** [comment]
I have been working on it a bit, having trouble getting the editions by publisher.

I did search before creating this issue but I agree it is a duplicate of #158

**Comment by RayBB** [comment]
As others have mentioned it is a duplicate so closing this one in favor of the original.
